### PR TITLE
publish return relevant data

### PR DESCRIPTION
**Pull request by Noviny**

### CODE REVIEW

**Comment by lukebatchelor** [comment]
Sweet! LGTM

**Comment by lukebatchelor** [comment]
Reckon we could/should explicitly state the return value shape of publish?

File: src/commands/publish.js [hunk 1]
```diff
@@ -106,7 +105,17 @@ export async function publish(opts: PublishOptions) {
       });
 
       if (publishConfirmation && publishConfirmation.published) {
-        publishedPackages.push({ name, version });
+        publishedPackages.push({
```
Comment (lukebatchelor) [inline src/commands/publish.js]: Could this be rewritten as

```
publishedPackages.push({
           name,
           newVersion: version,
           published: publishConfirmation && publishConfirmation.published
         });
```

Comment (Noviny) [reply src/commands/publish.js]: Yes. Good call.

**Pull request merged**